include/z64animation.h

```diff
@@ -88,41 +69,185 @@ typedef struct {
     /* 0x0C */ u16 staticIndexMax;
 } AnimationHeader; // size = 0x10
 
-// Unused
+/*
+ * SkelAnime
+ */
+
+typedef enum {
+    /* 0 */ ANIMMODE_LOOP,
+    /* 1 */ ANIMMODE_LOOP_INTERP,
+    /* 2 */ ANIMMODE_ONCE,
+    /* 3 */ ANIMMODE_ONCE_INTERP,
+    /* 4 */ ANIMMODE_LOOP_PARTIAL,
+    /* 5 */ ANIMMODE_LOOP_PARTIAL_INTERP
+} AnimationMode;
+
+typedef enum {
+    /* -1 */ ANIMTAPER_DECEL = -1,
+    /*  0 */ ANIMTAPER_NONE,
+    /*  1 */ ANIMTAPER_ACCEL
+} AnimationTapers;
+
+#define ANIM_FLAG_UPDATEY (1 << 1)
+#define ANIM_FLAG_NOMOVE (1 << 4)
```

Comment:
not your change, but i think UPDATE_Y and NO_MOVE would be an improvement

Reply:
skelanime moveFlags need a cleanup/documentation pass anyway, I can change this now but it'll probably be touched again in the near future anyway